salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -165,7 +165,8 @@
 							
 						}
 					},
-					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ] // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ], // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					removeButtons: ''
```

Comment:
this overwrites the default config, which comes from the file [config.js](https://github.com/dhlab-basel/Knora/blob/wip/update-ckeditor/salsah/src/public/vendor/ckeditor_4.7.0/config.js)